plugins/node/opentelemetry-instrumentation-redis/src/types.ts

```diff
@@ -17,16 +17,6 @@
 import { Span } from '@opentelemetry/api';
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
 
-// exported from
-// https://github.com/redis/node-redis/blob/v3.1.2/lib/command.js
-export interface RedisCommand {
-  command: string;
-  args: string[];
-  buffer_args: boolean;
-  callback: (err: Error | null, reply: unknown) => void;
-  call_on_write: boolean;
-}
```

Comment:
reviewer note:

this is technically a breaking change (since it was publicly exported by the package and now it is not). but it's only used internally and there is no reason for any user to consume it for anything.